spec/queries/get_incomplete_personal_statement_applications_ready_to_nudge_spec.rb

```diff
@@ -79,6 +99,7 @@
       submitted_at: nil,
       becoming_a_teacher_completed: false,
     )
+    create(:application_choice, application_form:)
     application_form.update_columns(
       updated_at: 10.days.ago,
     )
```

Comment:
Can we add the trait unsubmitted to represent the draft applications on this spec? Is it worth to add almost the same scenario but instead of creating the spec as draft applications we create submitted applications?